python/shotgun_data/shotgun_data_retriever.py

```diff
@@ -101,6 +102,7 @@ class ShotgunDataRetriever(QtCore.QObject):
     # thumbnails are downloaded last as they are considered low-priority 
     # and can take a relatively significant amount of time
     _DOWNLOAD_THUMB_PRIORITY        = 20
+    _DOWNLOAD_ATTACHMENT_PRIORITY   = 20
```

Comment:
same comment as above - attachment should have slightly higher prio than thumbs I think.